docs/modules/ROOT/pages/explanations/app_catalog.adoc

```diff
@@ -0,0 +1,118 @@
+Service provisioning is done via the OSB API, which behind the scenes operates on Kubernetes resources (same concept as above).
+This allows to integrate the service offering in Cloudfoundry, on external Kubernetes Cluster via the https://svc-cat.io/[Kubernetes Service Catalog] or on cloud providers which provide integration into their marketplace (for example https://community.exoscale.com/documentation/vendor/[Exoscale Marketplace]).
+
+== Providers
+
+https://crossplane.io/docs/master/concepts/providers.html[Crossplane Providers] implement the https://crossplane.io/docs/master/concepts/managed-resources.html[Managed Resources].
+Allthough they are all the same, we differentiate them:
+
+Cloud Services Provider::
+These providers do work with already existing service offerings from cloud providers like AWS or Google.
+The services are running in the cloud and are configured to our liking.
+
+Kubernetes-native Provider::
+The https://github.com/crossplane-contrib/provider-helm[Helm provider] installs and configures Helm Charts, either on the local Kubernetes Cluster or on remote Clusters.
+This also applies to the https://github.com/crossplane-contrib/provider-kubernetes[Kubernetes provider].
+These are our own custom services and are engineered and provided by VSHN.
+
+Configuration Provider::
+Some providers directly configure a service which was provisioned by another provider.
+For example https://github.com/crossplane-contrib/provider-sql[provider-sql] can configure users and permissions in an SQL database which was provisioned by the Helm Provider or by a Cloud Services Provider.
+
+== Configuration Management
+
+Crossplane itself is deployed and configured with the Project Syn https://github.com/projectsyn/component-crossplane[Commodore Component].
+Although there is the concept of https://crossplane.io/docs/master/concepts/packages.html[Crossplane packages], service definitions (XRDs and Compositions) and provider installation and configuration is delegated to Project Syn Commodore Components as well.
+
+== Quality of a Service
+
+Each service in the catalog must meet our quality requirements.
+For a high-level description, see https://products.docs.vshn.ch/products/appcat/index.html#_common_features[our product documentation].
+
+=== Functional
+
+* Provisioning of a service happens fully automated
+* Configuration of services is a declarative approach
+* Day-2 operations of a service is well defined and automated as far as possible
+
+=== Performance
+
+* Services are provisioned in under 1 minute
+* Every service defines its own performance requirements (answer time, reaction time, ...) and constraints
+
+=== Reliability
+
+* Services have a defined SLI (Service Level Indicator) and an appropriate SLO (Service Level Objective)
+* Backup of data is regularly conducted
```

Comment:
No word about if we monitor them or not. Could lead to wrong conclusions.

Reply:
Yes, I'm unsure where and how to write that.